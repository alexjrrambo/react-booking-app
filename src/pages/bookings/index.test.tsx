import { render } from "@testing-library/react";
import { describe, it, expect } from "vitest";
import { BookingsPage } from "./index";

describe("BookingsPage", () => {
  it("should match snapshot", () => {
    const { asFragment } = render(<BookingsPage />);
    expect(asFragment()).toMatchSnapshot();
  });
});
