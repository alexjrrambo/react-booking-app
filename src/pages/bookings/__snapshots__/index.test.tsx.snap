// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`BookingsPage > should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-dTvVRJ icqxif"
  >
    <div
      class="MuiPaper-root-bVLFAF chSBJf MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root-euZVcD bfnEXD MuiCard-root"
      style="--Paper-shadow: 0px 2px 1px -1px rgba(0,0,0,0.2),0px 1px 1px 0px rgba(0,0,0,0.14),0px 1px 3px 0px rgba(0,0,0,0.12);"
    >
      <div
        class="MuiCardContent-root-KYbLi jzgjqL MuiCardContent-root sc-jwTyAe fPeLhV"
      >
        <div
          class="sc-hjsuWn cdroja"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root-ljJYIa dsxfUI MuiSvgIcon-root MuiSvgIcon-fontSizeMedium"
            data-testid="HomeOutlinedIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="m12 5.69 5 4.5V18h-2v-6H9v6H7v-7.81zM12 3 2 12h3v8h6v-6h2v6h6v-8h3z"
            />
          </svg>
          <p
            class="MuiTypography-root-gsYkrY ccMIMr MuiTypography-root MuiTypography-body2"
          >
            Property 1
          </p>
        </div>
        <div
          class="sc-hjsuWn cdroja"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root-ljJYIa dsxfUI MuiSvgIcon-root MuiSvgIcon-fontSizeMedium"
            data-testid="CalendarMonthOutlinedIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M19 4h-1V2h-2v2H8V2H6v2H5c-1.11 0-1.99.9-1.99 2L3 20c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2m0 16H5V10h14zm0-12H5V6h14zM9 14H7v-2h2zm4 0h-2v-2h2zm4 0h-2v-2h2zm-8 4H7v-2h2zm4 0h-2v-2h2zm4 0h-2v-2h2z"
            />
          </svg>
          <p
            class="MuiTypography-root-gsYkrY ccMIMr MuiTypography-root MuiTypography-body2"
          >
            All dates
          </p>
        </div>
        <button
          class="MuiButtonBase-root-kHvgeY huicfx MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-colorPrimary MuiButton-root-jakOPJ gafVYE MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-colorPrimary"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiButton-startIcon-gVhczM hMbEGI MuiButton-icon MuiButton-startIcon MuiButton-iconSizeMedium"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root-ljJYIa dsxfUI MuiSvgIcon-root MuiSvgIcon-fontSizeMedium"
              data-testid="TuneOutlinedIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M3 17v2h6v-2zM3 5v2h10V5zm10 16v-2h8v-2h-8v-2h-2v6zM7 9v2H3v2h4v2h2V9zm14 4v-2H11v2zm-6-4h2V7h4V5h-4V3h-2z"
              />
            </svg>
          </span>
          Change filters
        </button>
      </div>
    </div>
    <button
      class="MuiButtonBase-root-kHvgeY huicfx MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-colorPrimary MuiButton-root-jakOPJ dFlcoc MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-colorPrimary"
      tabindex="0"
      type="button"
    >
      <span
        class="MuiButton-startIcon-gVhczM hMbEGI MuiButton-icon MuiButton-startIcon MuiButton-iconSizeMedium"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root-ljJYIa dsxfUI MuiSvgIcon-root MuiSvgIcon-fontSizeMedium"
          data-testid="AddIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6z"
          />
        </svg>
      </span>
      Create booking
    </button>
  </div>
</DocumentFragment>
`;
